elrond-wasm/src/api/send_api.rs

```diff
@@ -131,6 +106,46 @@ pub trait SendApi: ErrorApi + Clone + Sized {
 		)
 	}
 
+	/// Performs a simple ESDT/NFT transfer, but via async call.  
+	/// As with any async call, this immediately terminates the execution of the current call.  
+	/// So only use as the last call in your endpoint.  
+	/// If you want to perform multiple transfers, use `self.send().direct()` instead.  
+	/// Note that EGLD can NOT be transfered with this function.  
+	fn transfer_esdt_via_async_call(
+		&self,
+		to: &Address,
+		token: &TokenIdentifier,
+		nonce: u64,
+		amount: &Self::AmountType,
+		data: &[u8],
+	) -> ! {
+		if nonce == 0 {
```

Comment:
can't we combine EGLD transfer here as well ?

Reply:
Can EGLD be transferred with AsyncCall? Keep in mind this function doesn't call any endpoint, it performs a simple transfer.